packages/client/src/http-client/http-client.ts

```diff
@@ -4,8 +4,9 @@ import type { CustomDataType } from '@novu/shared';
 const DEFAULT_API_VERSION = 'v1';
 const DEFAULT_BACKEND_URL = 'https://api.novu.co';
 const PACKAGE_NAME = '@novu/client';
-const PACKAGE_VERSION = '0.42.0';
+const PACKAGE_VERSION = '2.0.0-canary.0';
 const DEFAULT_USER_AGENT = `${PACKAGE_NAME}-${PACKAGE_VERSION}`;
+const NOVU_API_VERSION = '2024-06-26';
```

Comment:
@BiswaViraj For now it's hardcoded and we just want to track post v2 requests, it's the date that Sokratis suggested

Reply:
Since we need the date mostly in Novu/js, should we allow the option on the client level and pass the date from the client instance used in `@novu/js`?

Reply:
If we need that date only in the `@novu/js` I would say to update the headers just in that package and not here.